src/tsconfig.app.json

```diff
@@ -12,7 +12,8 @@
   },
   "include": [
     "../src/**/*",
-    "../node_modules/angular-d3-graph/module.ts"
+    "../node_modules/angular-d3-graph/module.ts",
+    "/Users/lougro/repos/angular-d3-graph/module.ts"
```

Comment:
looks like an artifact from local testing, should be dropped before merge

Reply:
Ah, good catch. I'll send a fixup straightaway.